src/codegen/row_batch.cpp

```diff
@@ -196,6 +196,18 @@ RowBatch::RowBatch(CompilationContext &ctx, llvm::Value *tid_start,
                    llvm::Value *tid_end, Vector &selection_vector,
                    bool filtered)
     : context_(ctx),
+      tile_group_id_(nullptr),
+      tid_start_(tid_start),
+      tid_end_(tid_end),
+      num_rows_(nullptr),
+      selection_vector_(selection_vector),
+      filtered_(filtered) {}
+
+RowBatch::RowBatch(CompilationContext &ctx, llvm::Value *tile_group_id,
+                   llvm::Value *tid_start, llvm::Value *tid_end,
+                   Vector &selection_vector, bool filtered)
+    : context_(ctx),
+      tile_group_id_(tile_group_id),
```

Comment:
Change this so that the second constructor (without the tile_group_id) calls the first one (that does). Ensures initialization is consistent.